workbench-core/datasets-ui/src/pages/new.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { useProjects } from '@aws/workbench-core-accounts-ui';
+import { BaseLayout } from '@aws/workbench-core-swb-common-ui';
```

Comment:
wondering if workbench-core-swb-common-ui should be moved to workbench-core-common-ui. It would be better to keep this more generic

Reply:
concern is that datasets-ui now completely depends on swb-ui

Reply:
These all packages are all under `workbench-core` structure. Might be okay to move to workbench-core-common-ui (in fact renaming), however out of the scope of the current PR. 
@htztomic , please elaborate on dependency between datasets-ui and swb-ui

Reply:
the BaseLayout are part of the `@aws/workbench-core-swb-common-ui`. If datasets-ui is a common component, we would prefer to keep the dependency decoupled because other MA teams will onboard, they will depend on this package which means they would also transitively depend on the swb-ui component.

Reply:
That's true, but if you look at current list of dependencies for datasets-ui, it's already include reference to swb-common-ui. 
https://github.com/awslabs/solution-spark-on-aws/blob/develop/workbench-core/datasets-ui/package.json#L34

Confusion here could be that the name of the package should not be swb-common-ui, but common-ui, but this will be out of scope for this PR. I'll create ticket to rename swb-common-ui.